lms/djangoapps/philu_overrides/tests/test_context_processor.py

```diff
@@ -0,0 +1,590 @@
+"""
+Tests for context processors in philu_overrides
+"""
+
+import mock
+import pytest
+from django.conf import settings
+from django.test import RequestFactory
+from django.urls import reverse
+
+from lms.djangoapps.philu_overrides.constants import (
+    ACTIVATION_ALERT_TYPE,
+    ACTIVATION_ERROR_MSG_FORMAT,
+    ORG_DETAILS_UPDATE_ALERT,
+    ORG_OEF_UPDATE_ALERT
+)
+from lms.djangoapps.philu_overrides.context_processor import (
+    add_nodebb_endpoint,
+    get_cdn_link,
+    get_global_alert_messages
+)
+from student.tests.factories import AnonymousUserFactory, UserFactory
+
+
+@pytest.fixture()
+def request_object(db):  # pylint: disable=unused-argument
+    request = RequestFactory().get('/dummy_url')
+    request.user = UserFactory.create()
+    return request
+
+# Tests for get_global_alert_messages context processor
+
+# ACTIVATION_ERROR_MSG Tests
+
+
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.is_org_detail_platform_overlay_available', mock.Mock())
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.is_org_oef_prompt_available', mock.Mock())
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.get_org_oef_update_prompt', mock.Mock())
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.is_org_detail_prompt_available', mock.Mock())
+@mock.patch('lms.djangoapps.philu_overrides.context_processor.get_org_metric_update_prompt', mock.Mock())
+def test_get_activation_error_alert_message_success(request_object):  # pylint: disable=redefined-outer-name
```

Comment:
Let's please disable this at module level, instead of repeating it at 19 places.

Reply:
Or we can use `name` param of the fixure method
Copied from docs:
"name – The name of the fixture. This defaults to the name of the decorated function. If a fixture is used in the same module in which it is defined, the function name of the fixture will be shadowed by the function arg that requests the fixture; one way to resolve this is to name the decorated function fixture_<fixturename> and then use @pytest.fixture(name='<fixturename>')."